tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/MsTestV2EvpTests.cs

```diff
@@ -45,7 +45,7 @@ public void SubmitTraces(string packageVersion)
             var testModules = new List<MockCIVisibilityTestModule>();
             var expectedTestCount = version.CompareTo(new Version("2.2.5")) < 0 ? 13 : 15;
 
-            var sessionId = RandomIdGenerator.Shared.NextSpanId();
+            var sessionId = RandomIdGenerator.Shared.NextLegacyId();
```

Comment:
@tonyredondo: do these CI Vis values support the full range of uint64 values? or do they need to stay in blow int64.max?  (note uint64 vs int64)

Reply:
good question, will check with backend

Reply:
Looks like some of the tracer have done this for a while (unrelated to 128-bit trace ids), so it should be fine.